packages/react-server/src/render/render.tsx

```diff
@@ -64,7 +64,12 @@ export function createRender(render: RenderFunction) {
       });
     } catch (error) {
       logger.error(error);
-      throw error;
+      if (process.env.NODE_ENV === 'development') {
+        ctx.body = `Internal Server Errror: \n\n${error.message}`;
```

Comment:
I attempted to opt for something [cleaner](https://github.com/koajs/koa/wiki/Error-Handling#catching-downstream-errors), but couldn't get it to work.